app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -508,6 +511,21 @@ class SettingsFragment : PreferenceFragmentCompat() {
         }
     }
 
+    private fun updateFxAUseLocalMenu() {
+        val settings = requireContext().settings()
+        val preferenceUseLocalFxAServer =
+            findPreference<SwitchPreference>(getPreferenceKey(R.string.pref_key_use_local_fxa_server))
+        val enabled =
```

Comment:
Let's carry over comment from below:
`// Only enable changes to these prefs when the user isn't connected to an account.`